### PR TITLE
Allow different syscalls from kernels 5.12 -> 5.16

### PR DESCRIPTION
**Pull request by rumpl**
**- What I did**

Allowed different syscalls from kernels 5.12 -> 5.16

Kernel 5.12:

    mount_setattr: needs CAP_SYS_ADMIN

Kernel 5.14:

    quotactl_fd: needs CAP_SYS_ADMIN
    memfd_secret: always allowed

Kernel 5.15:

    process_mrelease: always allowed

Kernel 5.16:

    futex_waitv: always allowed

**- How I did it**

This is a carry from https://github.com/containerd/containerd/pull/6494

### CODE REVIEW

File: profiles/seccomp/default.json [hunk 1]
```diff
@@ -186,6 +187,9 @@
 				"landlock_add_rule",
 				"landlock_create_ruleset",
 				"landlock_restrict_self",
+				"landlock_add_rule",
+				"landlock_create_ruleset",
+				"landlock_restrict_self",
```
Comment (thaJeztah) [inline profiles/seccomp/default.json:192]: You can remove the landlock ones; those were just merged in https://github.com/moby/moby/pull/43199 (so are in there twice now 😅)

Comment (rumpl) [reply profiles/seccomp/default.json]: Done!

**Review by AkihiroSuda — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Another failure on Windows; unrelated, but in case there's an issue somewhere (I saw something similar recently);

```
=== RUN   TestRenameAnonymousContainer
    main_test.go:32: assertion failed: error is not nil: Error response from daemon: Could not kill running container 9000d33b282204b81b11913c1e49b3411b8f8d657a4f34899065ed48b08c62da, cannot remove - tried to kill container, but did not receive an exit event: failed to remove 9000d33b282204b81b11913c1e49b3411b8f8d657a4f34899065ed48b08c62da
    main_test.go:32: assertion failed: error is not nil: Error response from daemon: error while removing network: network network1TestRenameAnonymousContainer id ca59a08111066149cbc5b4e0a37cfc42e3d0f5b4fbbce9da1b33c85ba2a56490 has active endpoints: failed to remove network ca59a08111066149cbc5b4e0a37cfc42e3d0f5b4fbbce9da1b33c85ba2a56490
--- FAIL: TestRenameAnonymousContainer (23.53s)
```

<details>

```
[2022-05-13T17:44:26.275Z] INFO: Killing daemon with PID 11184
[2022-05-13T17:44:26.275Z] INFO: Tidying pidfile d:\CI\PR-43594\8\docker.pid
[2022-05-13T17:44:26.275Z] INFO: Stopping possible daemon pid 11184
[2022-05-13T17:44:26.275Z] ERROR: The process "11184" not found.
[2022-05-13T17:44:26.743Z] 
[2022-05-13T17:44:26.743Z] 
[2022-05-13T17:44:26.743Z] __GENUS          : 2
[2022-05-13T17:44:26.743Z] __CLASS          : __PARAMETERS
[2022-05-13T17:44:26.743Z] __SUPERCLASS     : 
[2022-05-13T17:44:26.743Z] __DYNASTY        : __PARAMETERS
[2022-05-13T17:44:26.743Z] __RELPATH        : 
[2022-05-13T17:44:26.743Z] __PROPERTY_COUNT : 1
[2022-05-13T17:44:26.743Z] __DERIVATION     : {}
[2022-05-13T17:44:26.743Z] __SERVER         : 
[2022-05-13T17:44:26.743Z] __NAMESPACE      : 
[2022-05-13T17:44:26.743Z] __PATH           : 
[2022-05-13T17:44:26.743Z] ReturnValue      : 0
[2022-05-13T17:44:26.743Z] PSComputerName   : 
[2022-05-13T17:44:26.743Z] 
[2022-05-13T17:44:26.743Z] __GENUS          : 2
[2022-05-13T17:44:26.743Z] __CLASS          : __PARAMETERS
[2022-05-13T17:44:26.743Z] __SUPERCLASS     : 
[2022-05-13T17:44:26.743Z] __DYNASTY        : __PARAMETERS
[2022-05-13T17:44:26.743Z] __RELPATH        : 
[2022-05-13T17:44:26.743Z] __PROPERTY_COUNT : 1
[2022-05-13T17:44:26.743Z] __DERIVATION     : {}
[2022-05-13T17:44:26.743Z] __SERVER         : 
[2022-05-13T17:44:26.743Z] __NAMESPACE      : 
[2022-05-13T17:44:26.743Z] __PATH           : 
[2022-05-13T17:44:26.743Z] ReturnValue      : 0
[2022-05-13T17:44:26.743Z] PSComputerName   : 
[2022-05-13T17:44:26.743Z] 
[2022-05-13T17:44:27.212Z] __GENUS          : 2
[2022-05-13T17:44:27.212Z] __CLASS          : __PARAMETERS
[2022-05-13T17:44:27.212Z] __SUPERCLASS     : 
[2022-05-13T17:44:27.212Z] __DYNASTY        : __PARAMETERS
[2022-05-13T17:44:27.212Z] __RELPATH        : 
[2022-05-13T17:44:27.212Z] __PROPERTY_COUNT : 1
[2022-05-13T17:44:27.212Z] __DERIVATION     : {}
[2022-05-13T17:44:27.212Z] __SERVER         : 
[2022-05-13T17:44:27.212Z] __NAMESPACE      : 
[2022-05-13T17:44:27.212Z] __PATH           : 
[2022-05-13T17:44:27.212Z] ReturnValue      : 0
[2022-05-13T17:44:27.212Z] PSComputerName   : 
[2022-05-13T17:44:27.212Z] 
[2022-05-13T17:44:27.212Z] __GENUS          : 2
[2022-05-13T17:44:27.212Z] __CLASS          : __PARAMETERS
[2022-05-13T17:44:27.212Z] __SUPERCLASS     : 
[2022-05-13T17:44:27.212Z] __DYNASTY        : __PARAMETERS
[2022-05-13T17:44:27.212Z] __RELPATH        : 
[2022-05-13T17:44:27.212Z] __PROPERTY_COUNT : 1
[2022-05-13T17:44:27.212Z] __DERIVATION     : {}
[2022-05-13T17:44:27.212Z] __SERVER         : 
[2022-05-13T17:44:27.212Z] __NAMESPACE      : 
[2022-05-13T17:44:27.212Z] __PATH           : 
[2022-05-13T17:44:27.212Z] ReturnValue      : 0
[2022-05-13T17:44:27.212Z] PSComputerName   : 
[2022-05-13T17:44:27.212Z] 
[2022-05-13T17:44:28.202Z] INFO: Nuking d:\CI
[2022-05-13T17:44:30.436Z] Exception 0xc0000420 0x0 0x0 0x7ff976472e0a
[2022-05-13T17:44:30.436Z] PC=0x7ff976472e0a
[2022-05-13T17:44:30.436Z] 
[2022-05-13T17:44:30.436Z] syscall.Syscall(0x7ff97125d430, 0x2, 0xc082069d68, 0xc0820066b0, 0x0, 0x3, 0x3, 0x0)
[2022-05-13T17:44:30.436Z] 	c:/go/src/runtime/syscall_windows.go:128 +0x5c
[2022-05-13T17:44:30.436Z] github.com/microsoft/hcsshim._destroyLayer(0xc082069d68, 0xc0820066b0, 0x0, 0x0)
[2022-05-13T17:44:30.436Z] 	e:/go/src/github.com/docker/docker/vendor/src/github.com/microsoft/hcsshim/zhcsshim.go:181 +0x9f
[2022-05-13T17:44:30.436Z] github.com/microsoft/hcsshim.destroyLayer(0xc082069d68, 0xc0820062cb, 0x2, 0x0, 0x0)
[2022-05-13T17:44:30.436Z] 	e:/go/src/github.com/docker/docker/vendor/src/github.com/microsoft/hcsshim/zhcsshim.go:174 +0x77
[2022-05-13T17:44:30.436Z] github.com/microsoft/hcsshim.DestroyLayer(0x0, 0xc0820062c8, 0x3, 0xc0820062cb, 0x2, 0x0, 0x0)
[2022-05-13T17:44:30.436Z] 	e:/go/src/github.com/docker/docker/vendor/src/github.com/microsoft/hcsshim/destroylayer.go:18 +0x38a
[2022-05-13T17:44:30.436Z] main.main()
[2022-05-13T17:44:30.436Z] 	E:/docker/ci/docker-ci-zap/zap.go:38 +0x23a
[2022-05-13T17:44:30.436Z] rax     0x20
[2022-05-13T17:44:30.436Z] rbx     0x0
[2022-05-13T17:44:30.436Z] rcx     0x60
[2022-05-13T17:44:30.436Z] rdi     0x20
[2022-05-13T17:44:30.436Z] rsi     0x13e48d0
[2022-05-13T17:44:30.436Z] rbp     0x8dc19
[2022-05-13T17:44:30.436Z] rsp     0x8dbb0
[2022-05-13T17:44:30.436Z] r8      0x20
[2022-05-13T17:44:30.436Z] r9      0x58
[2022-05-13T17:44:30.436Z] r10     0x17
[2022-05-13T17:44:30.436Z] r11     0x246
[2022-05-13T17:44:30.436Z] r12     0x0
[2022-05-13T17:44:30.436Z] r13     0x0
[2022-05-13T17:44:30.436Z] r14     0x0
[2022-05-13T17:44:30.437Z] r15     0x0
[2022-05-13T17:44:30.437Z] rip     0x7ff976472e0a
[2022-05-13T17:44:30.437Z] rflags  0x246
[2022-05-13T17:44:30.437Z] cs      0x33
[2022-05-13T17:44:30.437Z] fs      0x53
[2022-05-13T17:44:30.437Z] gs      0x2b
[2022-05-13T17:44:30.437Z] 
[2022-05-13T17:44:30.437Z] INFO: executeCI.ps1 exiting at Fri May 13 17:44:30 CUT 2022. Duration 00:24:56.6287914
[2022-05-13T17:44:30.437Z] 
```

</details>

**Comment by thaJeztah** [comment]
And Windows/c8d is hitting `TestSlowStdinClosing` again 🙄

**Comment by thaJeztah** [comment]
Bringing this one in

**Pull request merged**